### PR TITLE
[Merged by Bors] - add more info on state reduction to changelog

### PR DESCRIPTION
**Pull request by countvonzero**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ See [RELEASE](./RELEASE.md) for workflow instructions.
 
 ### Upgrade information
 
+This upgrade is incompatible with versions older than v1.1.6.
+
+At the start of the upgrade, mesh data will be pruned and compacted/vacuumed. Pruning takes longer for
+nodes that joined the network earlier. For 1-week-old nodes, it takes ~20 minutes. for 3-week-old
+nodes it takes ~40 minutes. Vacuum takes ~5 minutes.
```
Comment (dshulyak) [inline CHANGELOG.md]: please add info that during vacuum node needs x2 space

**Review by dshulyak — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,15 @@ See [RELEASE](./RELEASE.md) for workflow instructions.
 
 ### Upgrade information
 
+This upgrade is incompatible with versions older than v1.1.6.
+
+At the start of the upgrade, mesh data will be pruned and compacted/vacuumed. Pruning takes longer for
+nodes that joined the network earlier. For 1-week-old nodes, it takes ~20 minutes. for 3-week-old
+nodes it takes ~40 minutes. Vacuum takes ~5 minutes. The vacuum operation requires extra disk space
```
Comment (fasmat) [inline CHANGELOG.md]:
```suggestion
nodes that joined the network earlier. For 1-week-old nodes, it takes ~20 minutes. for 3-week-old
nodes it takes ~40 minutes. The vacuum operation requires extra disk space
```

That sentence seems out of place

**Review by fasmat — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (countvonzero) [reply CHANGELOG.md]: how so? i was trying to state that vacuum operation takes 5 minutes.

Comment (countvonzero) [reply CHANGELOG.md]: changed a little. hopefully its less awkward

Comment (fasmat) [reply CHANGELOG.md]: I was confused by the stated durations.

I assume pruning takes up to 40 minutes for 3 week old nodes and afterwards it takes another 5 minutes to vacuum independently of how old the node is? I wrongly read it as "takes 40 minutes to vacuum - vacuum takes 5 minutes". The updated text is clearer to me now 🙂

**Comment by countvonzero** [comment]
bors merge

**Pull request closed without merging**